discovery-provider/src/solana/anchor_program_indexer.py

```diff
@@ -133,4 +146,45 @@ def process_index_task(self):
     # parsed_transactions will contain an array of txs w/instructions
     # each containing relevant metadata in container
     async def fetch_ipfs_metadata(self, parsed_transactions):
-        return super().fetch_ipfs_metadata(parsed_transactions)
+
```

Comment:
One thing I wanted to do was while iterating over the parsed transactions was fetch all the ids for the user, track, playlist, etc. so the the db orm models can be fetched all at once. 
Some of that logic is in my PR. 
Thoughts on that?

Reply:
^we should do this - however won't that be in a step above this @jowlee ?

Reply:
isn't that later in `process_transactions` when we validate and save records?